src/lib_c/x86_64-windows-msvc/c/time.cr

```diff
@@ -0,0 +1,33 @@
+lib LibC
```

Comment:
Seems that all these definitions are in `winbase.h`, so perhaps they should be in `winbase.cr`?